repository_service_tuf/cli/admin/metadata.py

```diff
@@ -529,21 +527,21 @@ def update(
     upload: bool,
     run_ceremony: bool,
     save: bool,
-    upload_server: str,
+    api_server: str,
 ) -> None:
     """
     Start a new metadata update ceremony.
     """
     settings = context.obj["settings"]
     if upload and not run_ceremony:
         # Sever authentication or setup
-        if settings.AUTH and not upload_server:
+        if api_server is None:
             raise click.ClickException(
-                "Requires '--upload-server' when using '--auth'. "
-                "Example: --upload-server https://rstuf-api.example.com"
+                "Requires '--api-server' when using '--upload/-u'. "
+                "Example: --api-server https://api.rstuf.example.com"
```

Comment:
Why throw an error when the user could have setup `settings.SERVER` to be used meaning `api-url` is not required?
Also, `--api-server` is documented as `required=False`, so it should be optional.